harvest/api/paper.py

```diff
@@ -19,7 +19,7 @@ class PaperBroker(API):
 
     interval_list = ['1MIN', '5MIN', '15MIN', '30MIN', '1DAY']
 
-    def __init__(self, account_path: str=None):
+    def __init__(self, account_path: str=None, commission_fee: float=0.01):
```

Comment:
Commission fees can be fixed (like options on [Ameritrade](https://www.tdameritrade.com/pricing.html)), or based on a percentage (like [Gemini](https://www.gemini.com/fees/api-fee-schedule#section-api-fee-schedule)), so ideally there should be a way to switch between that. Additionally, brokers can have different brokerage fees depending on the type of asset.